fix for routine radio and checkbox width

bkdotcom opened the pull request:
fixes execute enum/set options and the "is deterministic" checkbox on edit

bkdotcom committed:
fix for routine radio and checkbox width

fixes execute enum/set options and the "is deterministic" checkbox on edit

lem9 commented:
Thanks. I tried with Chrome and Firefox the change for the checkbox; I see your intention but it's not working for me. Only the ".rte-table input" has an effect on this checkbox.

bkdotcom committed:
prior patch had typo ".rte-table" vs".rte_table"

typo... that's what I get for trying to make simple tweak via github website rather than desktop client

bkdotcom commented:
slop quick fix on my end...  i used `.rte-table` rather than `.rte_table`

lem9 commented:
Rebased and merged, thanks.
